source/docs/hardware-components/kit-and-hardware-guide/gobilda.rst

```diff
@@ -29,13 +29,22 @@ Notable components
 
    goBILDA :term:`U-channel <Channel>` (48x48 mm)
 
-- In addition, low-side channel (smaller profile :term:`channel <Channel>`) complements full size channel. Low side channel is just 12mm in height and allows for more compact builds. Additionally, low side channel opens up possibilities such as a kit-based :term:`parallel plate drivetrain <Parallel Plate Drivetrain>`. Low side channel also forms the basis of the cascading kit with a 2 or 4 stage option.
+- Low-side channel (smaller profile :term:`channel <Channel>`) complements full size channel. Low side channel is just 12mm in height and allows for more compact builds. Additionally, low side channel opens up possibilities such as a kit-based :term:`parallel plate drivetrain <Parallel Plate Drivetrain>`. Low side channel also forms the basis of the cascading kit with a 2 or 4 stage option.
```

Comment:
pick a hyphenation scheme for "low-side" and stick with it

is "parallel plate drivetrain" a proper noun?

Reply:
I don't think it is, its a description of a mechanism